pkg/vpcmodel/explainabilityPrint.go

```diff
@@ -257,21 +282,21 @@ func (rules *rulesConnection) ruleDetailsStr(c *VPCConfig, filtersRelevant map[s
 		ingressRulesDetails = rules.ingressRules.rulesDetailsStr(c, filtersRelevant, true)
 	}
 	if needEgress && egressRulesDetails != emptyString {
-		egressRulesDetails = "Egress:\n" + egressRulesDetails + newLine
+		egressRulesDetails = "\tEgress:\n" + egressRulesDetails + newLine
 	}
 	if needIngress && ingressRulesDetails != emptyString {
-		ingressRulesDetails = "Ingress:\n" + ingressRulesDetails + newLine
+		ingressRulesDetails = "\tIngress:\n" + ingressRulesDetails + newLine
 	}
 	return egressRulesDetails, ingressRulesDetails
 }
 
 // returns a string with the effect of each filter by calling StringFilterEffect
 // e.g. "security group sg1-ky allows connection; network ACL acl1-ky blocks connection"
-func (rulesInLayers rulesInLayers) summaryFiltersStr(c *VPCConfig, filtersRelevant map[string]bool, isIngress bool) string {
+func (rules rulesInLayers) summaryFiltersStr(c *VPCConfig, filtersRelevant map[string]bool, isIngress bool) string {
```

Comment:
"who is the receiver" comment

Reply:
same as above - *if* we accept this change then not in this PR